feature-flags.json

```diff
@@ -4,5 +4,6 @@
 	"dashboardSharing",
 	"ga4Reporting",
 	"gm3Components",
-	"userInput"
+	"userInput",
+	"enhancedMeasurement"
```

Comment:
Could we move it just below `dashboardSharing` to arrange it in alphabetical order?

Reply:
Order changed. Thanks. I thought latest flags go as last index in array.